### PR TITLE
DOC:  Grammatical fix in numpy doc

### PR DESCRIPTION
**Pull request by Ankeetshkk**

### CODE REVIEW

**Comment by mattip** [comment]
Could you rephrase that sentence to avoid the problem altogether?

**Comment by Ankeetshkk** [comment]
> Could you rephrase that sentence to avoid the problem altogether?

**Comment by mattip** [comment]
You can make a change, commit it, then push the change to your repo. It will automatically be pulled into to this PR.

**Comment by Ankeetshkk** [comment]
> You can make a change, commit it, then push the change to your repo. It will automatically be pulled into to this PR.

okay, when will be commit changes implemented in actual documentation.

**Comment by mattip** [comment]
The next release will be 1.17, the evolving documents are on https://numpy.org/devdocs.
Note there is another use of he/she on https://www.numpy.org/devdocs/reference/generated/numpy.random.RandomState.set_state.html, which is generated from https://github.com/numpy/numpy/blob/master/numpy/random/mtrand/mtrand.pyx#L726

**Comment by charris** [comment]
@mattip Rewritten.

**Review by eric-wieser — APPROVED** [review]

**Comment by mattip** [comment]
Thanks @Ankeetshkk, Chuck
